manifests/init.pp

```diff
@@ -21,9 +21,11 @@
   String                 $zones_template       = 'nsd/etc/nsd/nsd.zones.conf.erb',
   String                 $pattern_template     = 'nsd/etc/nsd/nsd.patterns.conf.erb',
   String                 $gather_template      = 'nsd/etc/nsd/nsd.gather.conf.erb',
+  String                 $includes_template    = 'nsd/etc/nsd/nsd.includes.conf.erb',
   Tea::Ip_address        $puppetdb_server      = '127.0.0.1',
   Tea::Port              $puppetdb_port        = 8080,
   Array[Tea::Ip_address] $ip_addresses         = [],
+  Array[String]          $includes             = [],
```

Comment:
String[1] would be better, TBH that wold be better everywhere but that's a different matter :)

Reply:
Thank you for the feedback.
fixed with https://github.com/icann-dns/puppet-nsd/pull/10/commits/74e5413ead6ed9917c0ab2944005c725fe84295d